.github/ISSUE_TEMPLATE/repo.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: Repo request
+description: Request a repository inside the `salt-extensions` organization for your project.
+title: "[Repo request]: "
+labels: [repo]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Welcome to the Salt extensions organization!
+        We're glad you want to host with us.
+
+  - type: dropdown
+    id: project_type
+    attributes:
+      label: Source of the code
+      description: Where do the modules originate from?
+      options:
+        - Salt core (migration)
+        - New extension
+    validations:
+      required: true
+
+  - type: input
+    id: project_name
+    attributes:
+      label: Project name
+      description: Tell us the name of the repository you're applying for.
+      placeholder: saltext-vault
+    validations:
+      required: true
+
+  - type: input
+    id: repo_url
+    attributes:
+      label: Repository URL
+      description: Where can we check out your Salt extension project?
+      placeholder: https://github.com/foo/saltext-bar
+    validations:
+      required: true
```

Comment:
I think this can be optional if a person wants to start from scratch. From the other side, having an existing repo is better to assess the code

Reply:
We could potentially make it optional with Trusted Publishing, agreed.